src/helpers.js

```diff
@@ -37,23 +55,23 @@ function normalizeAmount(amount) {
   return parseFloat(
     amount
       .replaceAll(/\s/, '')
+      .replace('"', '')
       .replace(',', '.')
       .trim()
   )
 }
 
 /**
- * Analyzes the CSS classes of the bank account to find its type
+ * Analyzes the URL of the bank account to find its type
  *
- * @param {string} label The CSS classes of the bank account
+ * @param {string} url The URL of the bank account
  * @see {@link https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.bank/#iocozybankaccounts|io.cozy.bank.accounts}
  * @returns {string} The type of the bank account
  */
-function getAccountTypeFromCSS(cssClasses) {
-  let cssClass = cssClasses.split(' ')
+function getAccountTypeFromUrl(url) {
+  let urlType = url.match(/\/compte\/(.+)\/[0-9a-f]+/)
```

Comment:
Doesn't work for credit card, because the URL is like this :  `/compte/cav/.../carte/...`

To fix that : 
```javascript
function getAccountTypeFromUrl(url) {
  let urlType = url.match(/\/compte\/(.+)\/[0-9a-f]+\/(\w+)?\/?/)
  let type = urlType[2] || urlType[1]

  return AbbrToAccountType[type] || 'Unknown'
}
```


This seems to work with other accounts. :)

Reply:
I had to tweak it a little bit, but I think I've got the correct regexp. Could you try again with your card account please?

Reply:
It works!